webv2/table/utilities.go

```diff
@@ -156,12 +154,52 @@ func updateTypeOfInterleaveColumn(interleaveTableSchema []InterleaveTableSchema,
 		}
 		interleaveTableSchema[tableIndex].InterleaveColumnChanges[colIndex].Type = previousType
 		interleaveTableSchema[tableIndex].InterleaveColumnChanges[colIndex].UpdateType = updateType
+		interleaveTableSchema[tableIndex].InterleaveColumnChanges[colIndex].Size = previousSize
+		interleaveTableSchema[tableIndex].InterleaveColumnChanges[colIndex].UpdateSize = newSize
 	}
 
 	return interleaveTableSchema
 }
 
-func createInterleaveColumnType(interleaveTableSchema []InterleaveTableSchema, tableIndex int, colIndex int, columnId string, colName string, previousType string, updateType string) []InterleaveTableSchema {
+func updateSizeOfInterleaveTableSchema(interleaveTableSchema []InterleaveTableSchema, table, columnId, colName, colType string, previousSize, newSize int) []InterleaveTableSchema {
+	tableIndex := isTablePresent(interleaveTableSchema, table)
+	interleaveTableSchema = createInterleaveTableSchema(interleaveTableSchema, table, tableIndex)
+	interleaveTableSchema = updateSizeOfInterleaveColumn(interleaveTableSchema, table, columnId, colName, colType, previousSize, newSize)
```

Comment:
This needs to be recursive.

Reply:
Done. Made the update size recursive along with all other updates to interleaved and foreign key references